resources/views/user/profile-edit.blade.php

```diff
@@ -33,13 +33,16 @@
     <div class="row justify-content-center">
       <div class="col-lg-4 offset-lg-sidebar">
         <div class="list-group" id="list-tab" role="tablist">
+          @if (FixometerHelper::hasRole(Auth::user(), 'Administrator') || Auth::id() == $user->id)
           <a class="list-group-item list-group-item-action active" id="list-profile-list" data-toggle="list" href="#list-profile" role="tab" aria-controls="profile">@lang('profile.profile')</a>
           <a class="list-group-item list-group-item-action" id="list-account-list" data-toggle="list" href="#list-account" role="tab" aria-controls="account">@lang('profile.account')</a>
 
           <a class="list-group-item list-group-item-action" id="list-email-preferences-list" data-toggle="list" href="#list-email-preferences" role="tab" aria-controls="email-preferences">@lang('profile.email_preferences')</a>
           <a class="list-group-item list-group-item-action" id="list-calendar-links-list" data-toggle="list" href="#list-calendar-links" role="tab" aria-controls="calendar-links">@lang('profile.calendars.title')</a>
-          @if (Auth::id() == $user->id)
-              <a class="list-group-item list-group-item-action" id="list-notifications-list" href="{{ route('notifications') }}" role="tab">@lang('profile.notifications')</a>
+          <a class="list-group-item list-group-item-action" id="list-notifications-list" href="{{ route('notifications') }}" role="tab">@lang('profile.notifications')</a>
+          @endif
+          @if(Auth::user()->isRepairDirectoryRegionalAdmin() || Auth::user()->isRepairDirectorySuperAdmin())
```

Comment:
Could this be in a policy also?  So we can call e.g. @can('viewRepairDirectorySettings') rather than having the same check inline here and below.

Reply:
Yes, it could and should be.  This is true of a zillion checks in the various blade templates.  

For this specific story, what do you want?

- Just this template
- All instances of this specific || test in other templates
- Policies for all related Auth checks in other templates

Reply:
For this specific story, just for this template.